standalone-metastore/metastore-catalog/data/conf/README.txt

```diff
@@ -0,0 +1 @@
+Need to force creation of a directory.
```

Comment:
nit: no data or configuration under the `data/conf`, can we remove it?

Reply:
yes, please remove it, if it does not contains anything. If later needs, you can put back anytime

Reply:
Ok.